global.d.ts

```diff
@@ -0,0 +1,10 @@
+declare namespace NodeJS {
```

Comment:
Isn't this defined by `@nuxt/types`?

Reply:
Yes. This is how we can extend it :)

Reply:
You are right i've added nuxt/types instead of our globals